### DISCUSSION TITLE
How to use useFetch with TS?

### DISCUSSION THREAD

**Discussion by francoism90** [vueuse/vueuse#1320] [Q&A]
How can you control the return type of let's say `data`?

Thanks!

**Comment by mihail727** [comment]
most direct way - you can use `as`:
```ts
const { data } = await useFetch(...) as { data: Ref<UserType> }
```
but in general, maybe there you can specify the expected type as in nuxt 3 try it:
```ts
const { data } = await useFetch<UserType>(...)
```
